doc/man3/ASN1_STRING_TABLE_add.pod

```diff
@@ -0,0 +1,65 @@
+=pod
+
+=head1 NAME
+
+ASN1_STRING_TABLE_add, ASN1_STRING_TABLE_get,
+ASN1_STRING_TABLE_cleanup - ASN1_STRING_TABLE manipulation functions
```

Comment:
You need to add `ASN1_STRING_TABLE` here.  A quick `make doc-nits` helps you see what's missing.